meta-resin-common/recipes-containers/docker/docker/docker.conf.systemd

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon --log-driver=journald -s btrfs -H fd:// -H tcp://0.0.0.0:2375/
```

Comment:
@floion we are working on a cli tool that will make it easier to create/deploy/update apps on a local, development ResinOS device. This tool will work by using the device Docker daemon for the aforementioned applications, so I guess that for v0 , and considering that this will be for development images only, it will be fine to use unencrypted/unauthenticated access.